sections/futures/PositionCard/PositionCard.tsx

```diff
@@ -21,380 +18,385 @@ import {
 	selectMarketAsset,
 	selectMarketKey,
 	selectPosition,
-	selectTradePreview,
-	selectFuturesType,
 	selectSkewAdjustedPrice,
 	selectMarketPriceInfo,
 	selectSelectedMarketPositionHistory,
+	selectPreviewData,
+	selectFuturesType,
 } from 'state/futures/selectors';
 import { useAppDispatch, useAppSelector } from 'state/hooks';
 import { PillButtonDiv } from 'styles/common';
 import media from 'styles/media';
 import { isFiatCurrency } from 'utils/currencies';
 import { formatDollars, formatPercent, zeroBN } from 'utils/formatters/number';
 import { formatNumber } from 'utils/formatters/number';
-import { getMarketName, getSynthDescription, isDecimalFour, MarketKeyByAsset } from 'utils/futures';
+import { getMarketName, isDecimalFour } from 'utils/futures';
 
 import EditLeverageModal from '../TradeCrossMargin/EditCrossMarginLeverageModal';
 
-type PositionCardProps = {
-	dashboard?: boolean;
-};
-
-type PositionData = {
-	marketShortName: string;
-	marketLongName: string;
-	marketPrice: JSX.Element;
-	positionSide: JSX.Element;
-	positionSize: string | React.ReactNode;
-	leverage: string | React.ReactNode;
-	liquidationPrice: string | JSX.Element;
-	pnl: string | Wei | JSX.Element;
-	realizedPnl: Wei;
-	pnlText: string | null | JSX.Element;
-	realizedPnlText: string | JSX.Element;
-	netFunding: Wei;
-	netFundingText: string | null | JSX.Element;
-	fees: string | JSX.Element;
-	avgEntryPrice: string | JSX.Element;
-};
-
-type PositionPreviewData = {
-	fillPrice: Wei;
-	sizeIsNotZero: boolean;
-	positionSide: string;
-	positionSize: Wei;
-	leverage: Wei;
-	liquidationPrice: Wei;
-	avgEntryPrice: Wei;
-	notionalValue: Wei;
-	showStatus: boolean;
-};
-
-const PositionCard: React.FC<PositionCardProps> = () => {
-	const { t } = useTranslation();
-	const { synthsMap } = Connector.useContainer();
-	const { selectedPriceCurrency } = useSelectedPriceCurrency();
-	const dispatch = useAppDispatch();
-
-	const futuresAccountType = useAppSelector(selectFuturesType);
-	const position = useAppSelector(selectPosition);
-	const marketAsset = useAppSelector(selectMarketAsset);
+const PositionCard = memo(() => {
 	const marketKey = useAppSelector(selectMarketKey);
-	const marketPrice = useAppSelector(selectSkewAdjustedPrice);
-	const previewTradeData = useAppSelector(selectTradePreview);
-	const thisPositionHistory = useAppSelector(selectSelectedMarketPositionHistory);
 	const openModal = useAppSelector(selectOpenModal);
 	const { isFuturesMarketClosed } = useFuturesMarketClosed(marketKey);
-	const marketPriceInfo = useAppSelector(selectMarketPriceInfo);
 
-	const positionDetails = position?.position ?? null;
+	return (
+		<>
+			{openModal === 'futures_cross_leverage' && <EditLeverageModal editMode="existing_position" />}
+
+			<Container id={isFuturesMarketClosed ? 'closed' : undefined}>
```

Comment:
Is the id needed?

Reply:
Apparently. Took it out earlier, but was surprised when I noticed line 466 & 477. In this vein though, I think it might make sense to port over the `useFuturesMarketClosed` hook to our SDK + Redux setup.